docs/examples/sample/README.md

```diff
@@ -0,0 +1,65 @@
+# How to use LeaderWorkerSet Features
+
+Below are some examples to use different features of LeaderWorkerSet.
+
+Deploy a LeaderWorkerSet with 3 groups and 4 workers in each group. You can find an example [here](lws.yaml)
+
+## Multi template for leader/worker pods
+
+LWS support using different templates for leader and worker pods. You could find the example [here](lws-multi-template.yaml),
+leader pod's spec is specified in leaderTemplate, and worker pods' spec is specified in workerTemplate.
+
+## Restart Policy
+
+You could specify the RestartPolicy to define the failure handling schematics for the pod group.
+By default, only failed pods will be automatically restarted. When the RestartPolicy is set to RecreateGroupOnRestart, it will recreate
+the groups on container/pods restarts. All the worker pods will be recreated after the new leader pod is started.
+You can find an example [here](lws-restart-policy.yaml).
```

Comment:
```suggestion
You can specify the RestartPolicy to define the failure handling schematics for the pod group.
By default, only failed pods will be automatically restarted. When the RestartPolicy is set to RecreateGroupOnRestart, it will recreate the pod group on container/pods restarts. All the worker pods will be recreated after the new leader pod is started. You can find an example [here](lws-restart-policy.yaml).
```